bigbluebutton-html5/imports/startup/server/index.js

```diff
@@ -154,24 +165,19 @@ WebApp.connectHandlers.use('/feedback', (req, res) => {
 
     if (!user) {
       Logger.error(`Feedback failed, user with id=${userId} wasn't found`);
```

Comment:
The log level should be lowered and the message changed to something like `Couldn't find user for feedback` since the feeback is still going to go through.

Reply:
Indeed, the message you suggested suits better the situation. But, what do you mean by "lower the message level"? would be not use `.error`?

Reply:
Warning level would be enough. It's not a high severity alert.